### DISCUSSION TITLE
The last day to register for Security Onion Conference 2021 is THIS Friday 9/24!

### DISCUSSION THREAD

**Discussion by dougburks** [Security-Onion-Solutions/securityonion#5587] [Announcements]
Please note that NO tickets will be sold at the door.  All attendees must pre-purchase their tickets.

https://securityonion.net/conference/

![soc2019](https://user-images.githubusercontent.com/1659467/134053689-4dec2436-83b3-4719-9b90-4ab7b6e4215f.jpg)

**Comment by presianbg** [comment]
Nice! 

Is the event going to be streamed online ?

**Reply by dougburks** [reply]
Talks will be recorded and uploaded to Youtube later.
